tools/webpack.config.ts

```diff
@@ -62,14 +80,59 @@ export default (env?: string) : webpack.Configuration => {
                         {loader: 'expose-loader', options: 'common'},
                     ],
                 },
+                // regular css files
+                {
+                    test: /\.css$/,
+                    use: [
+                        getCSSLoader(production),
+                        {
+                            loader: 'css-loader',
+                            options: {
+                                sourceMap: true,
+                                importLoaders: 1
+                            }
+                        },
+                    ],
+                },
+                // sass / scss loader
+                {
+                    test: /\.(sass|scss)$/,
+                    use: [
+                        getCSSLoader(production),
+                        {
+                            loader: 'css-loader',
+                            options: {
+                                sourceMap: true,
+                                importLoaders: 1
+                            }
+                        },
+                        {
+                            loader: 'sass-loader',
+                            options: {
+                                sourceMap: true
+                            }
+                        }
+                    ],
+                },
+                // load fonts and files
+                {
+                    test: /\.(woff(2)?|ttf|eot|svg|otf)(\?v=\d+\.\d+\.\d+)?$/,
+                    use: [{
+                        loader: 'file-loader',
+                        options: {
+                            name: '[name].[ext]',
+                            outputPath: 'fonts/'
+                        }
+                    }]
```

Comment:
Just to make sure I'm reading this correctly, block code is unused for now, right?

Reply:
This is being used by portico.scss which imports fonts via fonts.css